nucypher/cli/painting/help.py

```diff
@@ -41,35 +45,33 @@ def echo_solidity_version(ctx, param, value):
 def echo_config_root_path(ctx, param, value):
     if not value or ctx.resilient_parsing:
         return
-    click.secho(DEFAULT_CONFIG_ROOT)
+    click.secho(str(DEFAULT_CONFIG_ROOT.resolve()))
     ctx.exit()
 
 
 def echo_logging_root_path(ctx, param, value):
     if not value or ctx.resilient_parsing:
         return
-    click.secho(USER_LOG_DIR)
+    click.secho(str(USER_LOG_DIR.resolve()))
     ctx.exit()
 
 
 def paint_new_installation_help(emitter, new_configuration, filepath):
     character_config_class = new_configuration.__class__
     character_name = character_config_class.NAME.lower()
-
-    if new_configuration.keyring != NO_KEYRING_ATTACHED:
-        maybe_public_key = bytes(new_configuration.keyring.signing_public_key).hex()
+    if new_configuration.keystore != NO_KEYSTORE_ATTACHED:
+        maybe_public_key = new_configuration.keystore.id
```

Comment:
Seems that now it's not a public key at all.

Reply:
I admit the naming is a bit misleading here, but the Keystore's ID is set to the stamp's hex.